src/jdk.hotspot.agent/windows/native/libsaproc/sawindbg.cpp

```diff
@@ -561,7 +561,10 @@ static bool addThreads(JNIEnv* env, jobject obj) {
     #define REG_INDEX(x) sun_jvm_hotspot_debugger_amd64_AMD64ThreadContext_##x
 
     context.ContextFlags = CONTEXT_FULL | CONTEXT_DEBUG_REGISTERS;
-    ptrIDebugAdvanced->GetThreadContext(&context, sizeof(CONTEXT));
+    HRESULT hr = ptrIDebugAdvanced->GetThreadContext(&context, sizeof(CONTEXT));
+    if (hr != S_OK) {      
+      fprintf(stderr, "WARNING: GetThreadContext failed with hr=0x%x\n", hr);
```

Comment:
Consider filing an issue to fully investigate the best way to handle errors here, e.g. should this use the same error reporting structure as the other COM calls (COM_VERIFY_OK_)?